debian/rules

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@ --parallel
+	dh $@
 
 override_dh_auto_configure:
 	NOCONFIGURE=1 ./autogen.sh
 	dh_auto_configure -- --with-openssl --with-gnutls --without-cuse
 
-override_dh_usrlocal:
-
 override_dh_auto_test:
 	SWTPM_TEST_SECCOMP_OPT="--seccomp action=none" make -j4 check VERBOSE=1
+
+override_dh_makeshlibs:
+	dh_makeshlibs --no-scripts
```

Comment:
This keeps an `ldconfig(8)` trigger from being implicitly added by `debhelper` to the built package, which for `swtpm` was redundant (since it doesn't install any system-wide shared libraries) and was triggering Lintian warning `package-has-unnecessary-activation-of-ldconfig-trigger`.